prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -143,6 +148,18 @@ class FHIRConverter(
         }
     }
 
+    /**
+     * Loads a transformer schema with [schemaName] and returns it.
+     * Returns null if [schemaName] is the empty string.
+     * Using this function instead of calling the constructor directly simplifies the process of mocking the
+     * transformer in tests.
+     */
+    internal fun getTransformerFromSchema(schemaName: String): FhirTransformer? {
+        return if (schemaName != "") {
```

Comment:
Yea I suppose if for some reason it was set to whitespace other than `""`, we would still want to treat that as being blank rather than fail to read a file called `       `... will update this in an upcoming commit.